Tests/Data/PhaseField/beam3d_stag_1pcs.prj

```diff
@@ -0,0 +1,324 @@
+                    <type>Dirichlet</type>
+                    <component>1</component>
+                    <parameter>dirichlet0</parameter>
+                </boundary_condition>
+                <boundary_condition>
+                    <geometrical_set>beam3d_geometry</geometrical_set>
+                    <geometry>top</geometry>
+                    <type>Dirichlet</type>
+                    <component>1</component>
+                    <parameter>dirichlet0</parameter>
+                </boundary_condition>
+                <boundary_condition>
+                    <geometrical_set>beam3d_geometry</geometrical_set>
+                    <geometry>back</geometry>
+                    <type>Dirichlet</type>
+                    <component>2</component>
+                    <parameter>dirichlet0</parameter>
+                </boundary_condition>
+                <boundary_condition>
+                    <geometrical_set>beam3d_geometry</geometrical_set>
+                    <geometry>front</geometry>
+                    <type>Dirichlet</type>
+                    <component>2</component>
+                    <parameter>dirichlet0</parameter>
+                </boundary_condition>
+                <boundary_condition>
+                    <geometrical_set>beam3d_geometry</geometrical_set>
+                    <geometry>left</geometry>
+                    <type>Dirichlet</type>
+                    <component>0</component>
+                    <parameter>dirichlet2</parameter>
+                </boundary_condition>
+                <boundary_condition>
+                    <geometrical_set>beam3d_geometry</geometrical_set>
+                    <geometry>right</geometry>
+                    <type>Dirichlet</type>
+                    <component>0</component>
+                    <parameter>dirichlet1</parameter>
+                </boundary_condition>            
+            </boundary_conditions>
+        </process_variable>
+        
+    </process_variables>
+
+    <nonlinear_solvers>
+        <nonlinear_solver>
+            <name>basic_newton_d</name>
+            <type>Newton</type>
+            <max_iter>50</max_iter>
+            <linear_solver>linear_solver_d</linear_solver>
+        </nonlinear_solver>
+        <nonlinear_solver>
+            <name>basic_newton_u</name>
+            <type>Newton</type>
+            <max_iter>50</max_iter>
+            <linear_solver>linear_solver_u</linear_solver>
+        </nonlinear_solver>
+    </nonlinear_solvers>
+    <linear_solvers>
+        <linear_solver>
+            <name>linear_solver_d</name>
+            <eigen>
+                <solver_type>BiCGSTAB</solver_type>
+                <precon_type>ILUT</precon_type>
+            </eigen>
+        </linear_solver>
+        <linear_solver>
+            <name>linear_solver_u</name>
+            <eigen>
+                <solver_type>BiCGSTAB</solver_type>
+                <precon_type>ILUT</precon_type>
```

Comment:
tolerances and number of steps are missing.

Reply:
tolerances and number of steps have been added.